test/verify/check-storage-luks

```diff
@@ -377,5 +377,59 @@ class TestStorageLuks(StorageCase):
         self.content_row_wait_in_col(1, 1, "Encrypted data")
         self.content_row_wait_in_col(2, 1, "Unrecognized Data")
 
+    def testNoauto(self):
+        m = self.machine
+        b = self.browser
+
+        self.login_and_go("/storage")
+
+        # Add a disk and format it with luks and a filesystem, but without "Unlock at boot"
+
+        m.add_disk("50M", serial="MYDISK")
+        b.wait_in_text("#drives", "MYDISK")
+        b.click('tr:contains("MYDISK")')
+        b.wait_visible("#storage-detail")
+
+        self.content_head_action(1, "Format")
+        self.dialog({"type": "ext4",
+                     "crypto.on": True,
+                     "crypto_options.auto": False,
+                     "passphrase": "vainu-reku-toma-rolle-kaja",
+                     "passphrase2": "vainu-reku-toma-rolle-kaja",
+                     "mount_options.auto": True,
+                     "mount_point": "/run/foo"})
+        self.content_row_wait_in_col(1, 1, "Encrypted data")
+        self.content_tab_wait_in_info(1, 1, "Options", "noauto")
+        self.content_row_wait_in_col(2, 1, "ext4 File System")
+
+        # The filesystem should be mounted but have the "noauto" option
+        self.wait_mounted(2,1)
+        assert "noauto" in m.execute("grep /run/foo /etc/fstab || true")
```

Comment:
interesting, had no idea that this kind of assert is possible. But we use it a few times in verify tests indeed. Not sure if there is better/worse backtrace, but one can use `self.assertIn("noauto", m.execute("grep /run/foo /etc/fstab || true"))

Reply:
Yes, I'll change that to `self.assertIn`.

Reply:
Done.